Formula/jbig2dec.rb

```diff
@@ -5,14 +5,9 @@ class Jbig2dec < Formula
   sha256 "279476695b38f04939aa59d041be56f6bade3422003a406a85e9792c27118a37"
   license "AGPL-3.0-or-later"
 
-  # Not every GhostPDL release contains a jbig2dec archive, so we have to check
-  # the GitHub releases page instead (which we otherwise avoid). This is
-  # necessary because the jbig2dec homepage hasn't been updated to link to
-  # versions after 0.17.
   livecheck do
```

Comment:
```suggestion
  # Not every GhostPDL release contains a jbig2dec archive, so we have to check
  # the GitHub releases page (which we otherwise avoid) instead of the tags.
  # We avoid checking the jbig2dec homepage because it has been very slow to
  # update in the past when new versions were released.
  livecheck do
```